src/blocks.js

```diff
@@ -20,22 +22,25 @@
                 var count = parseInt(ctrl.count || 100);
 
                 ctrl.init({
-                    entity:'catalog-item',
-                    context:'search',
-                    filters:{
-                        type: 'uiBlocks',
+                    entity: 'catalog-item',
+                    context: 'search',
+                    filters: {
+                        type: ctrl.type,
                         partition: ctrl.partition
                     },
                     sortings: [
-                        {on:'priority', orientation:'desc'}
+                        {on: 'priority', orientation: 'desc'}
                     ],
-                    subset:{count: count},
-                    autosearch:true,
+                    subset: {count: count},
+                    autosearch: true,
                     noMoreResultsNotification: false
                 });
 
                 ctrl.templateUrl = 'partials/blocks' + ctrl.partition + 'blocks.html';
 
+                if (ctrl.readOnly != undefined) ctrl.active = false;
+                else ctrl.active = true;
```

Comment:
I recall a discussion earlier about not creating a new "active" attribute but simply testing for the existing "readOnly" one as it should resolve using truthy evaluation?

If not I'd remark that the following code does the same thing?
`ctrl.active = ctrl.readOnly == undefined`